src/views/dataaccesscoe/pl-planning.vue

```diff
@@ -0,0 +1,127 @@
+      </p>
+      <ul>
+        <li>
+          Advising on data access requirements clauses and solicitation language for Notice of Funding Opportunities (NOFOs), Requests for Proposals (RFPs) and internal ITS JPO-sponsored projects. We have standard language and clauses that can be tailored to your needs.
+        </li>
+        <li>
+          Determining applicability for data access requirements for your project, including whether data can be made public and what U.S. DOT-managed systems are be available to store project data.
+        </li>
+        <li>
+          Providing templates to begin data management in the solicitation phase (e.g. preliminary data management plans [DMPs] and instructions).
+        </li>
+        <li>
+          Other resources and support upon request. Contact us at
+          <a href="mailto:data.itsjpo@dot.gov">data.itsjpo@dot.gov<img class="in-line-external-link-icon" src="/images/icons/external-link_1b1b1b.svg" alt="External from DOT link icon" title="External link"></a>.
+        </li>
+      </ul>
+      <i>Resources</i>
+      <ul>
+        <li>
+          <router-link to="/resources/guidelines">ITS DataHub Guidelines</router-link> – these are ITS JPO’s overall guidelines and requirements for data access and management.
+        </li>
+        <li>
+          <router-link to="/resources/data-management/preliminary-dmp">Preliminary DMP</router-link> Instructions – a document to be used in a solicitation or early phases of a project to create a plan for data management. This document is filled out by a solicitation or project applicant.
+        </li>
+        <li>
+          Standard Solicitation Language – contact
+          <a href="mailto:data.itsjpo@dot.gov">data.itsjpo@dot.gov<img class="in-line-external-link-icon" src="/images/icons/external-link_1b1b1b.svg" alt="External from DOT link icon" title="External link"></a>
+          for standard data management solicitation language.
+        </li>
+        <li>
+          <router-link to="/resources/data-management/faqs">FAQs</router-link> – Frequently Asked Questions about the ITS JPO Guidelines and Resources.
+        </li>
+      </ul>
+    </div>
+
+    <div v-if="active_tab=='id_tab2'" id="id_tab2" class="usa-layout-docs__main desktop:grid-col-10 dh-data-access-coe_tab-content">
+      <p>
+        During the Planning phase, Data Providers should be brainstorming and planning project ideas. For help brainstorming potential project ideas and gathering lessons learned and cost estimates from previous ITS projects, see the
+        <a target="_blank" href="https://www.itsbenefits.its.dot.gov/its/benecost.nsf/ByLink/BenefitsAbout">ITS Benefits Database<img class="in-line-dot-link-new-tab" src="/images/icons/external-tabs.svg" alt="New tab icon." title="Opens in a new tab."></a>.
+        Additionally, review ITS JPO’s data management Resources page and associated templates and guidance to anticipate requirements that might be part of the project.
```

Comment:
`resources page` should be lowercase

Reply:
Disagree as Resource Page is a specific page on ITS DataHub